packages/design-system-tokens/src/tokens/font.ts

```diff
@@ -4,10 +4,10 @@
 import { FontTokens, to } from '../lib/types';
 
 const font = to<FontTokens>()({
-  'family-open-sans': "'Open Sans', Helvetica, sans-serif",
-  'family-rubik': "'Rubik', sans-serif",
-  'family-montserrat': "'Montserrat', sans-serif",
-  'family-bitter': 'Bitter, Georgia, serif',
+  'family-open-sans': "'Open Sans', Inter, Roboto, 'Helvetica Neue', 'Arial Nova', 'Nimbus Sans', Arial, sans-serif",
+  'family-rubik': "Rubik, Seravek, 'Gill Sans Nova', Ubuntu, Calibri, 'DejaVu Sans', source-sans-pro, sans-serif",
+  'family-montserrat': "Montserrat, Avenir, Corbel, 'URW Gothic', source-sans-pro, sans-serif",
```

Comment:
![geometric-humanist](https://github.com/CMSgov/design-system/assets/5159392/7ffd3dde-7dfb-4923-88fa-ae65bbc687d5)

Reply:
montserrat is a system font, so this was super easy